templates/response_radio.mustache

```diff
@@ -51,14 +51,17 @@
 {{#choices}}
     {{#horizontal}}<span style="white-space:nowrap;">{{/horizontal}}
     {{#selected}}
-    <span class="selected">
-        <input type="radio" name="{{name}}" checked="checked" />
-        {{{content}}}{{#othercontent}} <span class="response text">{{.}}</span>{{/othercontent}}
-    </span>
+        <span class="selected">
+            <input type="radio" aria-label="{{name}}" name="{{name}}" checked="checked" />
+                {{{content}}}{{#othercontent}} <span class="response text">{{.}}</span>{{/othercontent}}
+        </span>
     {{/selected}}
     {{^selected}}
-    <span class="unselected"><input type="radio" name="{{name}}" />
-        {{{content}}}</span>&nbsp;
+        <span class="unselected">
+            <input type="radio" aria-label="{{name}}" name="{{name}}" />
+                {{{content}}}
+            </span>
+        </label>&nbsp;
```

Comment:
Please double check this, both indentation is out and I don't see a closing tag for the label

Reply:
label tag removed, was added when testing with adding labels,